### ISSUE TITLE
Possible duplicate

### ISSUE THREAD

**Issue by ajrzepiela** [osc-em/OSCEM_Schemas#12]
In Instrument_schema.json:

  "Cryogen": {
                    "description": "Cryogen used in cooling the instrument and sample, usually nitrogen",
                    "type": "string"
                },

    "Holder_cryogen": {
                    "description": "Type of cryogen used in the holder - if the holder is cooled seperately",
                    "type": "string"
                },

**Comment by sbliven** [comment]
@sofyalaski which one gets used for mmcif?

**Comment by sofyalaski** [comment]
[This one:](http://mmcif.rcsb.org/dictionaries/mmcif_pdbx_v50.dic/Items/_em_imaging.cryogen.html) `_em_imaging.cryogen`.
The second one has no association with PDBx according to [conversions table](https://github.com/SwissOpenEM/LS_Metadata_reader/blob/2725784950de9474105ae24017d34f329f0be2d3/conversion/conversions.csv#L19)

**Comment by ytittes** [comment]
I agree, usually these two will be identical, so it might make sense to just consolidate them into one entry.
